jsonrpc/types/types.go

```diff
@@ -352,12 +352,14 @@ type Batch struct {
 	Blocks              []BlockOrHash       `json:"blocks"`
 	Transactions        []TransactionOrHash `json:"transactions"`
 	BatchL2Data         ArgBytes            `json:"batchL2Data"`
+	WIP                 bool                `json:"wip"`
```

Comment:
Do we really need both closed and wip fields? As far as I understood WIP is used exclusively to know if a batch is closed or not, in theory we don't need to add this field to the RPC response. It opens an opportunity to have inconsistent data in case closed and wip were equals for any reason. can we remove WIP and continue only with closed?

Reply:
then I proposed to remove 'closed' and left wip that is going to map the state.batch field